updater.go

```diff
@@ -295,8 +311,7 @@ type guiAppState struct {
 }
 
 func (u *Updater) checkUserActive(ctx Context) (bool, error) {
-
-	if u.guiBusyCount >= 3 {
+	if time.Duration(u.guiBusyCount)*u.tickDuration >= time.Hour*10 { // Allow the update through after 10 hours
```

Comment:
We used to wait max 3 hours, why the change to 10?

Reply:
I think we should leave it at 3

Reply:
Yeah, something around 3 sounds better to me too.  It's nice to know that an update we push with urgency will go to everyone online within 4 hours, instead of within 11.

Reply:
seemed less likely to trigger during a normal work day when the app frequently being used

Reply:
3 just means we will still all get app restarts throughout the day while using the app. the chance of needing an emergency update and the app being actively used for 11 hours seems slim, i strongly prefer leaving it on a longer window. the purpose of this check is really just in case we leave the busy file in an incorrect state so we eventually update, doesn't seem critical to have such a short time window.

Reply:
Well, it means we'll get likely one app restart in the day.  That's probably not a bad thing, I don't want to spend all work day still running yesterday's admin build if there's a new one out.

Reply:
This is only if you're actively using the app during all of these checks. I find it much more frustrating to have the app swept out from under me.